src/sql/util.py

```diff
@@ -156,6 +156,23 @@ def show_deprecation_warning():
     )
 
 
+def check_duplicate_arguments(args, cmd_from):
```

Comment:
It looks like we are using `cmd_from` argument only in the error message. I think we can remove this.

Reply:
It is now being used to access the dictionary elements to differentiate which arguments are allowed to be repeated. [Fixed here](https://github.com/ploomber/jupysql/pull/908/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R160)